### PR TITLE
chore(demo): add `Stackblitz` integration for documentation examples

### LINKED ISSUE #126: 📚 - Add `Stackblitz` integration for examples
### Which package(s) are relevant/related to the feature request?

Don't known / other

### Description

Screenshot 2023-02-15 at 13 13 04

### PR DESCRIPTION
**Pull request by nsbarsukov**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [X] Documentation content changes

## What is the current behavior?

Closes #126

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: projects/demo/src/pages/stackblitz/stackblitz.service.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    tuiRawLoad,
+    tuiTryParseMarkdownCodeBlock,
+} from '@taiga-ui/addon-doc';
+
+@Injectable()
+export class StackblitzService implements TuiCodeEditor {
+    private readonly baseProjectConfigs: Pick<Project, 'dependencies' | 'template'> = {
+        template: `typescript`,
+        dependencies: {
+            '@maskito/core': '*',
```
Comment (splincode) [inline projects/demo/src/pages/stackblitz/stackblitz.service.ts:14]: why not just use the fixed version from master?

image

Comment (nsbarsukov) [reply projects/demo/src/pages/stackblitz/stackblitz.service.ts:14]: We don't have many parrallel versions of maskito (so, we can take the latest one in stackblitz)

Comment (splincode) [reply projects/demo/src/pages/stackblitz/stackblitz.service.ts:14]: You don't have `MASKITO_VERSION` for runtime?

```ts
import { MASKITO_VERSION } from "@maskito/core";
```

Comment (splincode) [reply projects/demo/src/pages/stackblitz/stackblitz.service.ts:14]: aaa, okey

**Review by splincode — APPROVED** [review]

**Pull request merged**